Form/Type/SCDatetimeType.php

```diff
@@ -18,10 +18,10 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
-* DatetimeType
+* SCDatetimeType
 *
 */
-class DatetimeType extends AbstractType
+class SCDatetimeType extends AbstractType
```

Comment:
Porque daba un error de que DatetimeType ya existe.
Because it gives an error that Datetime Type already exists.